### ISSUE TITLE
Add functionality for distance fitting.

### ISSUE THREAD

**Issue by drvdputt** [BEAST-Fitting/beast#146]
I need this for my work on IC63. Our first thought is that there would just be a grid of distances, for which different models are run in parallel.

**Comment by karllark** [comment]
May be duplicate of #38.

**Comment by galaxyumi** [comment]
It seems like duplicate. We can work together on this. 
(Still waiting for ASTs for the SMASH data.)

**Comment by karllark** [comment]
Closing this issue.  Please use issue #38 to track this work.
